alert-common/src/main/java/com/synopsys/integration/alert/common/channel/MessageSplitter.java

```diff
@@ -0,0 +1,135 @@
+/**
+ * alert-common
+ *
+ * Copyright (c) 2019 Synopsys, Inc.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.synopsys.integration.alert.common.channel;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+//TODO use this for SlackChannel to create message chunks.
+public final class MessageSplitter {
```

Comment:
Do we want this as a component for DependencyInjection? We could pass the limit and lineSeparator when they're needed via the method being used. This seems kind of random to not be used as a component.

Reply:
We can evaluate that after the release.  This is implemented to initially fix the issue.